Taskfile.yaml

```diff
@@ -0,0 +1,39 @@
+version: 3
+
+# vars: # cannot be specified here due to https://github.com/go-task/task/issues/2108
+#   NESTED_MODULES: api
+#   API_DIRS: '{{.ROOT_DIR}}/api/core/v1alpha1/...'
+#   MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+#   CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/internal/... {{.ROOT_DIR}}/test/... {{.ROOT_DIR}}/api/core/v1alpha1/...'
+#   COMPONENTS: 'quota-operator'
+#   REPO_URL: 'https://github.com/openmcp-project/quota-operator'
+#   GENERATE_DOCS_INDEX: "true"
+
+includes:
+  shared:
+    taskfile: hack/common/Taskfile_controller.yaml
+    flatten: true
+    excludes: [] # put task names in here which are overwritten in this file
+    vars:
+      NESTED_MODULES: api
+      API_DIRS: '{{.ROOT_DIR}}/api/v1alpha1/...'
+      MANIFEST_OUT: '{{.ROOT_DIR}}/api/crds/manifests'
+      CODE_DIRS: '{{.ROOT_DIR}}/cmd/... {{.ROOT_DIR}}/api/v1alpha1/... {{.ROOT_DIR}}/api/install/... {{.ROOT_DIR}}/api/crds/...'
+      COMPONENTS: 'quota-operator'
+      REPO_URL: 'https://github.com/openmcp-project/quota-operator'
+      GENERATE_DOCS_INDEX: "true"
```

Comment:
We should either configure this so that it doesn't create an empty documentation index or disable it for now, see my comment here https://github.com/openmcp-project/project-workspace-operator/pull/36#discussion_r2068755836 for further information.